doc(release) release notes collect 24.04.18 gorgone 24.04.14 (#5069)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-os.mdx

## Centreon Collect

### 24.04.18

Release date: `February 25, 2026`

<details>
  <summary>Bug fixes</summary>

> - [Configuration] Fixed an issue where broker cache were not updated in some cases.

Reviewer: ```suggestion
- [Configuration] Fixed an issue where the broker cache would not be updated.
```

## Centreon Gorgone

### 24.04.14

Release date: `February 25, 2026`

<details>
  <summary>Bug fixes</summary>

> - [Packaging] Fix gorgoned logrotate configuration file.

Reviewer: ```suggestion
- [Packaging] Fixed an issue with the gorgoned logrotate configuration file.
```
